client/src/components/App.js

```diff
@@ -1,36 +1,39 @@
-import React, { useState, useEffect } from "react";
-import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
-import { hot } from "react-hot-loader/root";
+import React, { useState, useEffect } from "react"
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom"
+import { hot } from "react-hot-loader/root"
 
-import getCurrentUser from "../services/getCurrentUser";
-import "../assets/scss/main.scss";
-import RegistrationForm from "./registration/RegistrationForm";
-import SignInForm from "./authentication/SignInForm";
-import TopBar from "./layout/TopBar";
+import getCurrentUser from "../services/getCurrentUser"
+import "../assets/scss/main.scss"
+import RegistrationForm from "./registration/RegistrationForm"
+import SignInForm from "./authentication/SignInForm"
+import TopBar from "./layout/TopBar"
+
+import IndexPage from "./IndexPage"
 
 const App = (props) => {
-  const [currentUser, setCurrentUser] = useState(undefined);
+  const [currentUser, setCurrentUser] = useState(undefined)
   useEffect(() => {
     getCurrentUser()
       .then((user) => {
-        setCurrentUser(user);
+        setCurrentUser(user)
       })
       .catch(() => {
-        setCurrentUser(null);
-      });
-  }, []);
+        setCurrentUser(null)
+      })
+  }, [])
   return (
     <Router>
       <TopBar user={currentUser} />
       <Switch>
         <Route exact path="/">
           <h2>Hello from react</h2>
         </Route>
+        <Route exact path="/index" component={IndexPage} />
```

Comment:
Might want to name this path `/trips` since that is what is displayed at that route, `/index` is a bit obscure

`IndexPage` is not very descriptive as to what this component is for, consider something like `TripsList`

Reply:
Done